sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/RecognizePiiEntityAsyncClient.java

```diff
@@ -184,19 +190,28 @@
      * {@link RecognizePiiEntitiesResultCollection} from a {@link SimpleResponse} of {@link EntitiesResult}.
      *
      * @param documents The list of documents to recognize Personally Identifiable Information entities for.
-     * @param options The {@link TextAnalyticsRequestOptions} request options.
+     * @param options The additional configurable {@link RecognizePiiEntityOptions options} that may be passed when
+     * recognizing PII entities.
      * @param context Additional context that is passed through the Http pipeline during the service call.
      *
      * @return A mono {@link Response} that contains {@link RecognizePiiEntitiesResultCollection}.
      */
     private Mono<Response<RecognizePiiEntitiesResultCollection>> getRecognizePiiEntitiesResponse(
-        Iterable<TextDocumentInput> documents, TextAnalyticsRequestOptions options, Context context) {
+        Iterable<TextDocumentInput> documents, RecognizePiiEntityOptions options, Context context) {
+        String modelVersion = null;
```

Comment:
Just confirming, it is fine to send the modelVersion and domainFilter as null to the service?

Reply:
yes. By default.